Resources/Textures/_RMC14/Mobs/Slimes/chest.rsi/meta.json

```
@@ -0,0 +1,23 @@
+﻿{
+    "version": 1,
+    "license": "CC-BY-SA-4.0",
+    "copyright": "bone, core, lungs by Sha-Seng (GitHub). bubble and fishbowl by crocodilecarousel (github).",
```

Bubble and fishbowl aren't present in this rsi so it should be fine to omit
```suggestion
    "copyright": "bone, core, lungs by Sha-Seng (GitHub).",
```